ikasaneip/builder/jar/src/test/resources/logback-test.xml

```diff
@@ -4,10 +4,12 @@
     <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
             <charset>utf-8</charset>
-            <Pattern>[%p] %c - %m%n</Pattern>
+            <pattern>%r %d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
+            </pattern>
         </encoder>
     </appender>
-    <logger name="org.springframework" level="WARN"/>
+    <logger name="org.springframework" level="INFO"/>
```

Comment:
Please revert back to WARN, and Set Ikasan level on WARN as well.

Our builds often fail due to excesive logging

Reply:
done